libkbfs/interfaces.go

```diff
@@ -925,13 +930,17 @@ type DiskBlockCache interface {
 	// Delete deletes some blocks from the disk cache.
 	Delete(ctx context.Context, blockIDs []kbfsblock.ID) (numRemoved int,
 		sizeRemoved int64, err error)
-	// UpdateMetadata updates the LRU time to Now() for a given block.
+	// UpdateMetadata updates metadata for a given block in the disk cache.
 	UpdateMetadata(ctx context.Context, blockID kbfsblock.ID,
-		hasPrefetched bool) error
+		hasPrefetched, donePrefetch bool) error
```

Comment:
Yeah. I originally envisioned this as a single-level cache of the sync state of a given folder. But without the full depth it doesn't seem that helpful. WDYT? Should I spend some effort thinking about how to do `finishedPrefetch` in depth?